sqlx-core/src/postgres/options/mod.rs

```diff
@@ -260,6 +260,20 @@ impl PgConnectOptions {
         self
     }
 
+    /// Get a clone of the current database name.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # use sqlx_core::postgres::PgConnectOptions;
+    /// let options = PgConnectOptions::new()
+    ///     .database("postgres");
+    /// assert!(options.get_database().is_some());
+    /// ```
+    pub fn get_database(mut self) -> Option<String> {
+        self.database.clone()
+    }
+
```

Comment:
```suggestion
    /// Get the current database name.
    ///
    /// # Example
    ///
    /// ```rust
    /// # use sqlx_core::postgres::PgConnectOptions;
    /// let options = PgConnectOptions::new()
    ///     .database("postgres");
    /// assert!(options.get_database().is_some());
    /// ```
    pub fn get_database(&self) -> Option<&str> {
        self.database.as_deref()
    }

```